### PR TITLE
Include adults

### PR DESCRIPTION
**Pull request by ghost**
everyone deserves to be included!!!1!1!!one!!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -29,7 +29,12 @@ module.exports = function getReasonWhyKidsLoveTheTasteOfCinnamonToastCrunch() {
...
 }
 
-module.exports.iMeanMost =
+module.exports.andAdults =
+  function getReasonWhyKidsAndAdultsLoveTheTasteOfCinnamonToastCrunch() {
+    return "Don't stereotype"
+  }
+
+module.exports.andAdults.iMeanMost = module.exports.iMeanMost =
   function getReasonWhyMostKidsLoveTheTasteOfCinnamonToastCrunch() {
```
Comment (towerofnix) [inline index.js:38]: Should be `getReasonWhyMostKidsAndAdultsLoveTheTasteOfCinnamonToastCrunch`.

**Comment by quat1024** [comment]
Beware of scope creep! This feature might be better off in another NPM package

**Review by towerofnix — CHANGES_REQUESTED** [review]
Generally good; one nitpick. Note there'll need to be two functions (`getReasonWhyMostKidsLoveTheTasteOfCinnamonToastCrunch` and `getReasonWhyMostKidsAndAdultsLoveTheTasteOfCinnamonToastCrunch`), so you'll need to define those separately (no `module.exports.andAdults.iMeanMost = module.exports.iMeanMost = function …`)

Common strings (`"Don't stereotype"` and `"Because it's good"`) should probably be changed into constants. @quat1024 is wrong and scope creep is good; we should also add translation support-- Just kidding!

I need to read up on semantic versioning. Code written with 2.0.0 won't be broken with (this verson), but that doesn't go the other way around - code that depends on `andAdults` (and `andAdults.iMeanMost`) won't work with 2.0.0.

**Comment by towerofnix** [comment]
Actually uh, @quat1024 is right - the package name is `why-do-kids-love-the-taste-of-cinnamon-toast-crunch`. This should probably be made into a separate package. Relatedly, though, there should *definitely* be "child" packages (no pun intended) - why-do-people-love-the-taste-of-cinnamon-toast-crunch! :D

**Comment by ghost** [comment]
Haha too late I already got my hacktoberfest points :^)

**Comment by towerofnix** [comment]
that's kay, the point is to make pull requests, not to get them merged :^)

**Pull request closed without merging**